devicetypes/tommysqueak/danfoss-living-connect-radiator-thermostat.src/danfoss-living-connect-radiator-thermostat.groovy

```diff
@@ -44,65 +45,73 @@ metadata {
 	}
 
 	// http://scripts.3dgo.net/smartthings/icons/
-	//	TODO: create temp set like Nest thermostat - http://docs.smartthings.com/en/latest/device-type-developers-guide/tiles-metadata.html
+	// TODO: create temp set like Nest thermostat - http://docs.smartthings.com/en/latest/device-type-developers-guide/tiles-metadata.html
 	tiles(scale: 2) {
+
 		multiAttributeTile(name:"richtemp", type:"thermostat", width:6, height:4) {
 			tileAttribute("device.nextHeatingSetpoint", key: "PRIMARY_CONTROL") {
-				attributeState("default", label:'${currentValue}°',
-					backgroundColors:[
-						// Celsius Color Range
-						[value: 0, color: "#153591"],
-						[value: 7, color: "#1e9cbb"],
-						[value: 15, color: "#90d2a7"],
-						[value: 23, color: "#44b621"],
-						[value: 29, color: "#f1d801"],
-						[value: 33, color: "#d04e00"],
-						[value: 36, color: "#bc2323"],
-						// Fahrenheit Color Range
-						[value: 40, color: "#153591"],
-						[value: 44, color: "#1e9cbb"],
-						[value: 59, color: "#90d2a7"],
-						[value: 74, color: "#44b621"],
-						[value: 84, color: "#f1d801"],
-						[value: 92, color: "#d04e00"],
-						[value: 96, color: "#bc2323"]
-					]
-				)
+				attributeState "default", label:'${currentValue}°', unit:"dC"
 			}
 
 			tileAttribute("device.nextHeatingSetpoint", key: "VALUE_CONTROL") {
-				attributeState("VALUE_UP", action: "temperatureUp")
-				attributeState("VALUE_DOWN", action: "temperatureDown")
+				attributeState "VALUE_UP", action: "tempUp"
+				attributeState "VALUE_DOWN", action: "tempDown"
+			}
+
+			tileAttribute("device.battery", key: "SECONDARY_CONTROL") {
+				attributeState "default", label:'${currentValue}%', unit:"%", icon:"st.nest.empty" // TODO: update battery icon
```

Comment:
TIL I like this trick with the empty icon. I remember trying this previously and getting a waterdrop icon, when I didn't specify an icon.